Assignments/Program.cs

```diff
@@ -1,3 +1,202 @@
-﻿using System;
+﻿// ---------------------------------------------------------------------------------------
+// Academy23 Assignments
+// Copyright (c) Metamation India.
+// ---------------------------------------------------------------------------------------
+// Program.cs
+// A12 - Wordle Game
+// ---------------------------------------------------------------------------------------
 
-Console.WriteLine("Hello, World!");
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.IO;
+using static System.ConsoleColor;
+using System.Reflection;
+
+namespace Academy;
+public class A12 {
+   public static void Main () => new Wordle ().PlayTheGame ();
+}
+
+public class Wordle {
+   public string secretWord, text;
+   public int triesCount = 0, ithComment = 0;
+   public bool testing = false, gameOver;
+   public int originX, originY;               // Reference X & Y coordinates.
+   public List<List<char>> word = new (6);    // List To display and store user entered letters.
+   readonly List<char> grayList = new (),     // List to collect incorrect letters.
+                       blueList = new (),     // List to collect correct but misplaced letters.
+                       greenList = new ();    // List to collect correct and correctly placed letters.
+   public string[] dictWords, guessWords;     // To store loaded strings
+   public StreamWriter testOutputFile = null; // Created for testing purpose.

```

Comment:
Why are these fields public? Moreover, naming conventions are not followed.

Reply:
The reason for making the fields as public is to use those fields in the MSTest class.

Reply:
@vikrampg1710-tmm: Exposing all these fields is not the correct design. Think of a better way to do it. May be, in the testing mode some things like the secret word can be hardcoded etc.